common/djangoapps/startup_configurations/validate_config.py

```diff
@@ -9,34 +9,17 @@ def validate_lms_config(settings):
     """
     validate_common_config(settings)
 
-    # validate feature based configurations
-    validate_marketing_site_config(settings)
-
 
 def validate_cms_config(settings):
     """
     Validates configurations for lms and raise ValueError if not valid
     """
     validate_common_config(settings)
 
-    # validate feature based configurations
-    validate_marketing_site_config(settings)
-
 
 def validate_common_config(settings):
     """
     Validates configurations common for all apps
     """
     if not getattr(settings, 'LMS_ROOT_URL', None):
         raise ValueError("'LMS_ROOT_URL' is not defined.")
-
-
-def validate_marketing_site_config(settings):
```

Comment:
@rlucioni I introduced this as part of this PR. If you look at earlier commit you will see that we made use of `MKTG_URLS` and so we introduced its validation on startup.
But now we are not using `MKTG_URLS` and i am not sure if i should still include its validation as part of this PR